docs/security/mitigations/mitigating-memcached-amplification-attacks.md

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
Mitigations is missing `_index.md` and will not be visible from the navigation pages

Reply:
✔️